File: server/aspnet/nvslonlineApi/nvslonlineApi/Models/Season.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace nvslonlineApi.Models
{
    public class Season
    {
        public int Id { get; set; }

        public string SeasonName { get; set; }

        public bool Active { get; set; }

        public bool IsHidden { get; set; }

        public DateTime SeasonStart { get; set; }

        public DateTime SeasonEnd { get; set; }
    }
}

File: client/angular2/NvslOnlineAngular2App/src/app/shared/models/season.ts
import { ISeason } from '../interfaces/interfaces';

export class Season implements ISeason{    
    
    constructor (public Id: number, public SeasonName: string, public Active: boolean, public IsHidden: boolean, public SeasonStart: string, public SeasonEnd: string ) {
        this.Id = Id;
        this.SeasonName = SeasonName;
        this.Active = Active;
        this.IsHidden = IsHidden;
        this.SeasonStart = SeasonStart;
        this.SeasonEnd = SeasonEnd;
    } 
}
